AccountTransferApi/app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion '26.0.0'
+    compileSdkVersion 30
+    buildToolsVersion '30.0.3'
     defaultConfig {
         applicationId "com.google.accounttransfer.sample"
         minSdkVersion 19
-        targetSdkVersion 26
+        targetSdkVersion 29
```

Comment:
Why targeting 29 if you compile it with 30?

Reply:
done (not special purpose, some 30, some 29, random)